app/presenters/chat_channel_detail_presenter.rb

```diff
@@ -39,4 +40,9 @@ def membership_users(memberships)
       }
     end
   end
+
+  def invitation_link(chat_channel)
+    host = Rails.application.config.app_domain
+    "#{host}/chat_channel_memberships/join_channel_invitation/#{chat_channel.slug}"
```

Comment:
Instead of `Rails.application.config.app_domain` it may be better if we use [URL.url](https://github.com/thepracticaldev/dev.to/blob/master/app/lib/url.rb#L13) here to maintain consistency and reliability on urls. It can then look something like this:

```
slug = chat_channel.slug
URL.url("/chat_channel_memberships/join_channel_invitation/#{slug}")
```